src/utilities/customNumberFormat.ts

```diff
@@ -0,0 +1,28 @@
+
+    /**
+     *  Replace '.'  with ','  
+     *  Add space between thousands.
+     * @param value the value to be formatted. 
+     */
+   export function numberFormat(value:number){
+        return new Intl.NumberFormat("nb-NO").format(value);
+    }
```

Comment:
I think these two functions could be simplified to a single function:
```suggestion
export function numberFormat(value: number, decimals: number | undefined) {
    const fractionDigits = decimals ? { minimumFractionDigits: decimals, maximumFractionDigits: decimals } : undefined;
    return new Intl.NumberFormat("nb-NO", fractionDigits).format(value);
}
```

Reply:
Yes good suggestion. The 'decimals' parameter can be optional.